src/lib.rs

```diff
@@ -68,15 +68,26 @@ pub fn deserialize_dynamic_slippage_config() -> Result<(DefaultSlippage, Vec<Cat
 
 #[cfg(test)]
 mod tests {
+    use itertools::Itertools;
+
     use super::*;
 
     #[test]
     fn test_deserialize_dynamic_slippage_config() {
         let (_, categories) = deserialize_dynamic_slippage_config().unwrap();
 
         assert_eq!(
-            categories.into_iter().map(|c| c.name).collect::<Vec<_>>(),
+            categories.iter().map(|c| &c.name).collect::<Vec<_>>(),
             vec!["stable", "lst", "bluechip", "verified"]
         );
+
+        // Ensure no duplicated mints, otherwise the config behaviour will be confusing
```

Comment:
it is checking within the remote list if there are any duplicates, and if there are, the CI fails, correct?